tests/app/test_party_service.py

```diff
@@ -22,83 +22,106 @@ def setUp(self):
 
-        self.assertTrue(mock_request.call_count == 1)
+        self.assertTrue(results, texts)
 
     @requests_mock.mock()
-    def test_results_returned_from_get_business_get_executed_twice_for_different_ru(self, mock_request):
+    def test_multiple_business_uuid_one_invalid(self, mock_request):
+        """Test get business details sends a request and returns data"""
+        ru_ids = ["c614e64e-d981-4eba-b016-d9822f09a4fb", "12345"]
+        texts = "error"
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}&id={ru_ids[1]}"
+        mock_request.get(business_data_url, status_code=400, reason="Invalid uuid", text=texts)
+        with self.app.app_context():
+            results = PartyService().get_business_details(ru_ids)
+
+        self.assertEqual(results, [])
+
+    @requests_mock.mock()
+    def test_multiple_business_uuid_one_not_found(self, mock_request):
         """Test get business details sends a request and returns data"""
-        ru = "1234"
+        ru_ids = ["c614e64e-d981-4eba-b016-d9822f09a4fb", "c614e64e-d981-4eba-b016-d9822f09a4f2"]
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}&id={ru_ids[1]}"
+        mock_request.get(business_data_url, status_code=200, text='{"something": "else"}')
+        with self.app.app_context():
+            results = PartyService().get_business_details(ru_ids)
 
-        sut = PartyService()
+        self.assertEqual(results, {"something": "else"})
+
+    @requests_mock.mock()
+    def test_results_returned_from_get_business_get_executed_twice_for_different_ru(self, mock_request):
+        """Test get business details sends a request and returns data"""
+        ru_ids = ["1234"]
         count = 0
         for retries in range(0, 2):
             count += 1
-            ru += str(count)
-            business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses/id/{ru}"
+            ru_ids += str(count)
+            business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}"
             mock_request.get(business_data_url, status_code=200, reason="OK", text='{"something": "else"}')
-
             with self.app.app_context():
-                sut.get_business_details(ru)
+                PartyService().get_business_details(ru_ids)
 
         self.assertTrue(mock_request.call_count == 2)
 
     @requests_mock.mock()
     def test_results_returned_from_get_business_details_returned_as_expected(self, mock_request):
         """Test get business details sends a request and returns data"""
-        ru = "1234"
-        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses/id/{ru}"
+        ru_ids = ["b08c07c3-df28-4283-bb4c-c048729ce372"]
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}"
```

Comment:
Similar to previous comments, it would be clearer if we just checked the mock was called rather that checking for `text` that we've mocked.